From 5c62bdce291258ea9788be263dadbf1bf3d8a8c9 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Wed, 21 Nov 2018 16:25:44 +0200
Subject: [PATCH 5174/5242] MLK-20328-2: ARM: dts: imx7ulp: adjust the
 expected format

commit  61056cf0ca0b36d3cc63c123210e3f64be59e821 from
https://source.codeaurora.org/external/imx/linux-imx.git

Default value must be prefixed by "0".

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
(cherry picked from commit 4ce702331a6c7b5a84fe4fdc210b62147e04fa84)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk-wm8960.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7ulp-evk-wm8960.dts b/arch/arm/boot/dts/imx7ulp-evk-wm8960.dts
index 73df08a..e3598f5 100644
--- a/arch/arm/boot/dts/imx7ulp-evk-wm8960.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk-wm8960.dts
@@ -195,6 +195,6 @@
 			  <&clks_m4 IMX7ULP_CM4_CLK_SAI0_DIV>;
 	assigned-clock-parents = <&clks_m4 IMX7ULP_CM4_CLK_APLL_PFD0_PRE_DIV>;
 	assigned-clock-rates = <0>, <12288000>;
-	fsl,dataline = <0x1 0x1>;
+	fsl,dataline = <0 0x1 0x1>;
 	status = "okay";
 };
-- 
1.7.9.5

